src/core/core.helpers.js

```diff
@@ -791,13 +795,14 @@ module.exports = function(Chart) {
 	};
 	helpers.retinaScale = function(chart) {
 		var ctx = chart.ctx;
-		var width = chart.canvas.width;
-		var height = chart.canvas.height;
+		var canvas = chart.canvas;
```

Comment:
`chart.canvas` or `ctx.canvas` (might be the same, but modified lines 799 & 800 use `ctx.canvas`)